lib/cc/analyzer/container.rb

```diff
@@ -75,13 +75,8 @@ def run(options = [])
         )
       ensure
         t_timeout.kill if t_timeout
```

Comment:
I think doing that would be an issue if something raised an error before one of them was initialized (which is the purpose of the `if t_out` right?)